src/cli/self_update.rs

```diff
@@ -282,8 +282,9 @@ tools:
 
     https://visualstudio.microsoft.com/visual-cpp-build-tools/
 
-Please ensure the Windows 10 SDK and the English language pack components
-are included when installing the Visual C++ Build Tools.
+Check its box for "Desktop development with C++" which will ensure that
+the Windows 10 SDK is selected.  If your locale language is not English,
+then additionally check the box for English under Language packs.  Or minimally, use the tab Single Components and select these items: “MSVC - VS 2019 C++ x64/x86 build tools” and “Windows 10 SDK”
```

Comment:
I just noticed that you used unicode characters `“..”` for the left and right quote. This is inconsistent with the rest of the text that uses the ASCII `".."`